components/connection/AddEdit.tsx

```diff
@@ -191,9 +204,25 @@ const AddEdit = ({ connection }: AddEditProps) => {
   const [{ status }, setSaveStatus] = useState<{ status: 'UNKNOWN' | 'SUCCESS' | 'ERROR' }>({
     status: 'UNKNOWN',
   });
+
   const saveConnection = async (event) => {
     event.preventDefault();
-    const { rawMetadata, redirectUrl, oidcDiscoveryUrl, oidcClientId, oidcClientSecret, ...rest } = formObj;
+    const {
+      rawMetadata,
+      redirectUrl,
+      oidcDiscoveryUrl,
+      oidcClientId,
+      oidcClientSecret,
+      metadataUrl,
+      ...rest
+    } = formObj;
+
+    if (Boolean(rawMetadata) === Boolean(metadataUrl) || (metadataUrl && !metadataUrl.startsWith('https'))) {
```

Comment:
If both `rawMetadata` and `metadataUrl` are present, it is showing an error. We could consider metadataUrl as the source of truth in that case.